.github/workflows/release-connect-npm-dependencies.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           submodules: recursive
 
       - name: Deploy to NPM ${{ github.event.inputs.package }}
-        uses: ./.github/workflows/template-release-connect-npm.yml
+        uses: ./.github/actions/release-connect-npm
```

Comment:
this should be `uses: ./.github/actions/release-connect-npm@develop` for us

Reply:
Done https://github.com/trezor/trezor-suite/pull/12103/files#diff-7350b81fd8fe25cd59872c938153a0d92081665fe4e6ee69e63ab0a7a69e78d5R37